src/widget/findondiscogs.cpp

```diff
@@ -0,0 +1,37 @@
+#include "findondiscogs.h"
+
+#include <QMenu>
+#include <QtDebug>
+
+#include "track/track.h"
+#include "util/parented_ptr.h"
+
+namespace {
+const QString kServiceTitle = QStringLiteral("Discogs");
+
+const QString kSearchUrl = QStringLiteral(
+        "https://www.discogs.com/search/?"); // List of Query needed.
+} //namespace
+
+void FindOnDiscogs::addSubmenusForServices(QMenu* pFindOnMenu, const Track& track) {
```

Comment:
This can become: 
```suggestion
void FindOnDiscogs::FindOnDiscogs(QMenu* pFindOnMenu, const Track& track) {
```

Reply:
I think that's better to use constructors right? Now I have created the service menus in two ways, I am not sure which one is better for the codebase tho. I have pushed the changes. First I have used FindOnWeb's method addsubmenusforservices, second I have used a constructor on FindOnSoundcloud.